CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 6.4.11
+### Fixed
+- Fixed so that `AuthManager` clears the previous refresh timer before setting a new one (thanks, @cranberyxl!)
```

Comment:
Rephrase probably?

Reply:
Rephrase how?

Reply:
Fixed `a component` such that.. 
Or 
`Authmanager` now clears.... 
`Something` fixed so that...

Reply:
Isn't that literally what I wrote?

Reply:
😅haha.. ok nvm.. on the first read, felt like a word is missing .. 'like a Bug X was fixed so that...'